relm4/src/lib.rs

```diff
@@ -51,6 +51,7 @@ pub use app::RelmApp;
 pub use tokio::task::JoinHandle;
 pub use util::{WidgetPlus, WidgetRef};
 
+use gtk::prelude::*;
 use once_cell::sync::OnceCell;
```

Comment:
Are there so many imports that it's worth to use the wildcard here?

Reply:
It could go either way, but for a prelude import I usually prefer to just glob import it so that all the convenient types are already in scope.

Reply:
It*s surely more convenient but IMO the code is easier to read when you know which types are in scope and from which import they come from.